diff --git a/build.gradle.kts b/build.gradle.kts
index 06bb1e6..1e1530d 100644
--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -11,6 +11,7 @@ plugins {
 android {
     namespace = "com.ai.assistance.operit.terminal"
     compileSdk = 36
+    ndkVersion = "29.0.14206865"
 
     defaultConfig {
         minSdk = 26
diff --git a/src/main/jniLibs/arm64-v8a/libsudo.so b/src/main/jniLibs/arm64-v8a/libsudo.so
deleted file mode 100644
index 6f1ea3e..0000000
--- a/src/main/jniLibs/arm64-v8a/libsudo.so
+++ /dev/null
@@ -1 +0,0 @@
-$@
\ No newline at end of file
